Add `Duration.inIntUnits` (#1)
eygraber/kotlin-duration-extensions: README.md

2.days
```

> There are also extension properties supplied on `Duration` to get the underlying value as a `Double` abd as `Int`:

Reviewer: ```suggestion
There are also extension properties supplied on `Duration` to get the underlying value as a `Double` and as an `Int`:
```

In addition, the `Duration` properties that returned the value as a `Double` have been deprecated.  This is because `Duration` is now backed by a `Long`.

The new way to retrieve the value as a `Double` is to use `Duration.toDouble(DurationUnit)`. That can be a little verbose, so this library provides `Duration.inDouble*` functions.

> Since a lot of existing APIs accept and `Int` amount of milliseconds/seconds/etc, so this library also provides `Duration.inWholeInt*` functions.

Reviewer: ```suggestion
Since a lot of existing APIs accept an `Int` amount of milliseconds/seconds/etc, this library also provides `Duration.inWholeInt*` functions.
```
